### DISCUSSION TITLE
Karma test explorer very unstable

### DISCUSSION THREAD

**Discussion by shivabayar** [lucono/karma-test-explorer#107] [Issues / Help]
The explorer crashes most of the time on VS code. 
VS code Version Details:
Version: 1.87.2
Commit: 863d2581ecda6849923a2118d93a088b0745d9d6
Date: 2024-03-08T15:20:57.520Z (5 mos ago)
Electron: 27.3.2
ElectronBuildId: 26836302
Chromium: 118.0.5993.159
Node.js: 18.17.1
V8: 11.8.172.18-electron.0
OS: Darwin arm64 23.5.0

Plugin version: v0.8.2

Also when re-running a test case it always complains that a operation is in progress. 
The plugin rarely works. Sorry for the feedback.

**Comment by MJFVerschuur** [comment]
When it works, it;'s amazing. Just what you need.
But running once and then running again gives the error 'Another test operation is still running'

karma test explorer version: 0.8.2
VSCode version details: 
Version: 1.99.3 (system setup)
Commit: 17baf841131aa23349f217ca7c570c76ee87b957
Date: 2025-04-15T23:18:46.076Z
Electron: 34.3.2
ElectronBuildId: 11161073
Chromium: 132.0.6834.210
Node.js: 20.18.3
V8: 13.2.152.41-electron.0
OS: Windows_NT x64 10.0.19045
